app/templates/direct-award/save-search.html

```diff
@@ -33,36 +33,68 @@
   </div>
   <form method="post" action="{{ url_for('direct_award.project_create', framework_framework=framework_framework)}}" id="createProjectForm">
     <div class="grid-row">
-      <div class="column-two-thirds dmspeak">
+      <div class="column-two-thirds">
         <div class="panel panel-border-wide">
           {% with content = search_summary_sentence %}
             {% include "toolkit/search-summary.html" %}
           {% endwith %}
         </div>
-        <h2 class="heading-xmedium">Create a new saved search</h2>
-        <p>Name your search. A short description of what you want to buy makes a good name.</p>
-        {% if form.name.errors %}
-        <div class="validation-wrapper">
-            {% endif %}
-            <div class="question" id="{{ form.name.name }}">
+
+        <div class="dmspeak">
+          <h2 class="heading-xmedium">Choose where to save your search</h2>
+        </div>
+        
+        <fieldset>
+          <div class="multiple-choice" data-target="new-search">
+            <input type="radio" name="save_search_selection" value="new_search" 
+              {% if request.form.save_search_selection == 'new_search' or not projects %}checked="checked"{% endif %} 
+              required="required">
+            <label for="save_search_selection">Create a new search</label>
+          </div>
+          <div class="panel panel-border-narrow js-hidden" id="new-search">
+            {% if form.name.errors %}
+            <div class="validation-wrapper">
```

Comment:
The approach I tend to prefer for this is to leave the `div` in place regardless of whether there are errors, and wrap just the attribute/class name in the `if`. That means you don't have to worry about having another `if` at some random further point down the page, which makes the indentation look all horrible too :)

Reply:
This part was copied over from the existing code. Can refactor if it's an issue?